packages/account/src/Components/account-limits/account-limits.scss

```diff
@@ -59,13 +59,10 @@
 
         &-wrapper {
             flex: 1;
+            margin-right: 1rem;
```

Comment:
Can u check if this works - `margin-inline-end: 1rem;`

Reply:
works

Reply:
Just need to make sure, if you are making some style into logical css prop then incase if some other css prop overrides it on some condition, make sure the overriding css is also logical css. Else, it will not overrride and dir-rtl or ltr style will get applied by default.